### PR TITLE
Add the "To Your Homes" and "First Water Works" Sections 

### PR DESCRIPTION
**Pull request by ellenbechtel**
Issue #8 
- Added the "Getting Water To Your Homes" and "First Water Works" sections
- Accidentally did a commit in the wrong place first, so there's an extra set of commits that were overwritten and that you can ignore.

### CODE REVIEW

File: dryville_story.md [hunk 1]
```diff
@@ -15,3 +15,17 @@ So let's say you made the big move. You and some friends have found your (desola
...
+
+But you still have the problem of keeping the storage tank full. Lugging water up to it is no better than lugging it up to your house; so you need to find a way to pump the water from the creek uphill into the storage tank. Since Dryville doesn't have any industry to produce products to sell to the outside world to bring in money to buy things with yet, you need to build your own pump. And you can't use electricity since you haven't built a power plant yet (but you will). Can you think of a source of energy that can run your pump? A windmill could do the job, but in the desert there isn't always wind. Your creek! The water in your creek is flowing nicely -- so you build a paddle wheel in the river. The paddle spins in the creek's current and turns a rod that runs your homemade pump. Voila! You've got water being pumped uphill into your storage tank where gravity lets it flow to your homes.
+
+##Dryville's First Water Works
```
Comment (lindsayplatt) [inline dryville_story.md]: For Markdown, you need to put a space between `##` and your words. If you look at the rendered version of this, the title is missing the right formatting.

image

**Review by lindsayplatt — CHANGES_REQUESTED** [review]
It's interesting, it appears that you committed from two different accounts. 

image

File: dryville_story.md (hunk 1 above)
Comment (ellenbechtel) [reply dryville_story.md]: Ah yes, dumb mistake. It's fixed!

**Comment by ellenbechtel** [comment]
Re: two accounts making edits - yes, I accidentally made the first round of edits inside githubs web interface.  When I checked my local files with `git status` there were no changes, so I realized I had done it in the wrong place and repeated the edit for practice in Notepad.  Both commits were recorded though - does that make a difference?

Re: edits in markdown - done!

**Comment by lindsayplatt** [comment]
No, I don't think that it matters. Just strange that your user looks different when it is coming from your computer - I believe they should be the same since your github account is linked to your computer (that whole SSH keys thing).

**Review by lindsayplatt — APPROVED** [review]
🎊

**Pull request merged**